.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: android
 
-groovy:
```

Comment:
@benjamin-bader The "groovy" block in the `.travis.yml` file is for installing `groovy` on the build and using it locally(eg. groovy file.groovy). We are using Groovy as a language through Gradle. I do not think you need to install `groovy` locally in order to compile Groovy with Gradle unlike Java. You need to have Java installed to run and compile with Gradle. I could be wrong about Groovy.